reco_utils/azureml/wide_deep.py

```diff
@@ -49,6 +49,7 @@ def _log(metric, value):
 parser.add_argument('--datastore', type=str, dest='DATA_DIR', help="Datastore path")
 parser.add_argument('--train-datapath', type=str, dest='TRAIN_PICKLE_PATH')
 parser.add_argument('--test-datapath', type=str, dest='TEST_PICKLE_PATH')
+parser.add_argument('--model-dir', type=str, dest='MODEL_DIR', default='model_checkpoints')
```

Comment:
Have you checked this works when called from the AML Hyperdrive notebook?

Reply:
I haven't but I hope it should work, why shouldn't?

Reply:
This is for the checkpoint dir clean up (not sure if aml needs this or not though).
If you want to see tensorboard, you should store the model_checkpoints under the output folder so that you can download later and open it from tensorboard.

Reply:
this should work, the default behavior is exactly what we had before, but now we can add a different name